sdks/python/apache_beam/utils/proto_utils_test.py

```diff
@@ -0,0 +1,25 @@
+import unittest
```

Comment:
RAT check wants you to add a license here, see other files.

Reply:
Thank you will fix.